testsuite/tests/mgc/test_basic.py

```diff
@@ -30,7 +32,9 @@ def test_smoke(route):
     Tests whether the backend, exposed using the HTTPRoute and Gateway, was exposed correctly,
     having a tls secured endpoint with a hostname managed by MGC
     """
-    backend_client = route.client(verify=False)  # self-signed certificate; TBD
+    backend_client = HttpxBackoffClient(
```

Comment:
can we verify that certificate is correct somehow? I think additional asserts, or HTTPX verify would be great